httplib.h

```diff
@@ -3833,6 +3835,18 @@ inline MultipartFormData Request::get_file_value(const char *key) const {
   return MultipartFormData();
 }
 
+/**
+ * Return multiple values corresponding to the key
+*/
+std::vector<MultipartFormData> Request::get_file_mutil_value(const char *key) const {
```

Comment:
Could you add `inline` keyword at the beginning of the line as other methods do?